docs/core/tools/dotnet-test.md

```diff
@@ -54,213 +33,103 @@ Test projects specify the test runner using an ordinary `<PackageReference>` ele
+- **`c|--configuration {Debug|Release}`**
 
-`-s|--settings <SETTINGS_FILE>`
+  Defines the build configuration. The default value is `Debug`, but your project's configuration could override this default SDK setting.
 
-The `.runsettings` file to use for running the tests. [Configure unit tests by using a `.runsettings` file.](/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file)
+- **`-collect <DATA_COLLECTOR_FRIENDLY_NAME>`**
 
-`-t|--list-tests`
+  Enables data collector for the test run. For more information, see [Monitor and analyze test run](https://aka.ms/vstest-collect).
 
-List all of the discovered tests in the current project.
+- **`d|--diag <PATH_TO_DIAGNOSTICS_FILE>`**
 
-`-v|--verbosity <LEVEL>`
+  Enables diagnostic mode for the test platform and write diagnostic messages to the specified file.
 
-Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
+- **`f|--framework <FRAMEWORK>`**
 
-`RunSettings arguments`
+  Looks for test binaries for a specific [framework](../../standard/frameworks.md).
 
-Arguments passed as RunSettings configurations for the test. Arguments are specified as `[name]=[value]` pairs after "-- " (note the space after --). A space is used to separate multiple `[name]=[value]` pairs.
+- **`--filter <EXPRESSION>`**
 
-Example: `dotnet test -- MSTest.DeploymentEnabled=false MSTest.MapInconclusiveToFailed=True`
+  Filters out tests in the current project using the given expression. For more information, see the [Filter option details](#filter-option-details) section. For more information and examples on how to use selective unit test filtering, see [Running selective unit tests](../testing/selective-unit-tests.md).
 
-For more information about RunSettings, see [vstest.console.exe: Passing RunSettings args](https://github.com/Microsoft/vstest-docs/blob/master/docs/RunSettingsArguments.md).
+- **`h|--help`**
 
-# [.NET Core 2.0](#tab/netcore20)
+  Prints out a short help for the command.
 
-`-a|--test-adapter-path <PATH_TO_ADAPTER>`
+- **`l|--logger <LoggerUri/FriendlyName>`**
 
-Use the custom test adapters from the specified path in the test run.
+  Specifies a logger for test results.
 
-`-c|--configuration {Debug|Release}`
+- **`--no-build`**
 
-Defines the build configuration. The default value is `Debug`, but your project's configuration could override this default SDK setting.
+  Doesn't build the test project before running it. It also implicitly sets the - `--no-restore` flag.
 
-`--collect <DATA_COLLECTOR_FRIENDLY_NAME>`
+- **`--no-restore`**
 
-Enables data collector for the test run. For more information, see [Monitor and analyze test run](https://aka.ms/vstest-collect).
+  Doesn't execute an implicit restore when running the command.
 
-`-d|--diag <PATH_TO_DIAGNOSTICS_FILE>`
+- **`o|--output <OUTPUT_DIRECTORY>`**
```

Comment:
I just noticed these singles are missing the `-` at the beginning. May want to look for that in the other files too.

Reply:
Good catch, thanks -- must have been an errant global find-and-replace.